.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": false,
```

Comment:
I was initially unsure about this setting given most of the JS/TS-centric codebases I've seen prefer single-quotes for some reason... But this is in alignment with Octopus Server's portal settings, so LGTM.